packages/babel-plugin-check-es2015-constants/src/index.js

```diff
@@ -44,7 +44,15 @@ export default function({ messages, types: t }) {
                 ]),
               );
             } else if (violation.parentPath.isForXStatement()) {
-              violation.parentPath.insertBefore(throwNode);
+              // Transform single statement body into BlockStatement
+              if (!violation.parentPath.get("body").isBlockStatement()) {
```

Comment:
Ah cool! As far as I can tell it uses this converter function, so it is probably appropriate:

https://github.com/babel/babel/blob/4e50b2d9d9c376cee7a2cbf56553fe5b982ea53c/packages/babel-types/src/converters.js#L243-L259

I will give it a try, thanks!